examples/spring-boot-webflux/src/main/java/example/springframework/boot/webflux/HelloConfiguration.java

```diff
@@ -24,21 +33,42 @@
      */
     @Bean
     public ArmeriaServerConfigurator armeriaServerConfigurator() {
-        // An example which adds a LoggingService decorator to the ServerBuilder in order to log every request.
-        return builder -> builder.decorator(LoggingService.newDecorator())
-                                 .accessLogWriter(AccessLogWriter.combined(), false);
+        // Customize the server using the given ServerBuilder. For example:
+        return builder -> {
+            // Add DocService that enables you to send Thrift and gRPC requests from web browser.
+            builder.serviceUnder("/docs", new DocService());
+
+            // Logging every request and response which is received by the server.
+            builder.decorator(LoggingService.newDecorator());
+
+            // Write access log after completing a request.
+            builder.accessLogWriter(AccessLogWriter.combined(), false);
+
+            // You can also bind asynchronous RPC services such as Thrift and gRPC:
+            // builder.service(THttpService.of(...));
+            // builder.service(new GrpcServiceBuilder()...build());
+        };
     }
 
     /**
      * A user can configure a {@link Client} by providing an {@link ArmeriaClientConfigurator} bean.
      */
     @Bean
     public ArmeriaClientConfigurator armeriaClientConfigurator() {
-        // An example which sets a custom client factory to the HttpClientBuilder in order not to validate
-        // a certificate received from the server.
-        return builder -> builder.factory(
-                new ClientFactoryBuilder()
-                        .sslContextCustomizer(b -> b.trustManager(InsecureTrustManagerFactory.INSTANCE))
-                        .build());
+        // Customize the client using the given HttpClientBuilder. For example:
+        return builder -> {
+            // Use circuit breaker for every endpoint.
+            final Function<String, CircuitBreaker> factory = key -> CircuitBreaker.of("my-cb-" + key);
+            final CircuitBreakerStrategy strategy = CircuitBreakerStrategy.onServerErrorStatus();
+            builder.decorator(CircuitBreakerHttpClient.newPerHostDecorator(factory, strategy));
```

Comment:
We can just do `builder.decorator(new CircuitBreakerHttpClientBuilder(strategy).build())` and remove the `factory` part up there. :)

Reply:
Thank you! I didn't notice the default `CircuitBreakerMapping`. 😆